zio-http/src/test/scala/zhttp/service/ServerSpec.scala

```diff
@@ -157,8 +161,14 @@ object ServerSpec extends HttpRunnableSpec(8088) {
         for {
           data <- status(!! / "success").repeatN(1024)
         } yield assertTrue(data == Status.OK)
+      } +
+      testM("Multiple Websocket Requests") {
+        val p = (for {
+          res <- send(basicRequest.get(uri"ws://localhost:8088/subscriptions").response(asWebSocketUnsafe))
+        } yield res.code).repeatN(1024)
+        assertM(p)(equalTo(StatusCode(101)))
 
-      }
+      }.provideCustomLayer(AsyncHttpClientZioBackend.layer().orDie)
   }
```

Comment:
- Create a separate `WebSocketServerSpec.scala` and move the tests there.
- Ensure no tests changes are required when ZIO-Http Client supports websockets.